src/System.Text.Primitives/System/Text/Formatting/Internal/IntegerFormatter.cs

```diff
@@ -2,32 +2,33 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
```

Comment:
Nope. Artifact of me doing the change. Initially the IsHexFormat helper was in this file, then I moved it outside. I have it removed in another PR that is coming in about 30 minutes.